applications/IgaApplication/custom_python/iga_python_application.cpp

```diff
@@ -46,6 +46,16 @@ PYBIND11_MODULE(KratosIgaApplication, m)
 
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DEAD_LOAD)
 
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, PK2_STRESS)
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, CAUCHY_STRESS)
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, CAUCHY_STRESS_TOP)
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, CAUCHY_STRESS_BOTTOM)
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, MEMBRANE_FORCE)
+    KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(m, INTERNAL_MOMENT)
+
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, SHEAR_FORCE_1)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, SHEAR_FORCE_2)
+
```

Comment:
please make sure this is consistent with StructuralMechanics
=> Check the variable-name used here: https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h